libs/@local/hash-graph-sdk/python/graph_sdk/filter/base.py

```diff
@@ -0,0 +1,69 @@
+from abc import ABC
+from typing import Generic, Self, TypeVar
+
+from graph_client import QueryToken
+from graph_client.models import Selector
+
+from graph_sdk.query import Path
+
+
+class AbstractPath(ABC):
+    """An abstract path."""
+
+    path: Path
+
+    def __init__(self) -> None:
+        """Initialize the filter with a path."""
```

Comment:
This does not initialize a filter (We have `Filter` in the Graph so this is misleading)

Reply:
fixed in [69155df](https://github.com/hashintel/hash/pull/2878/commits/69155dfc220c339740d314c66f7f295ba3261685)